discord/threads.py

```diff
@@ -121,6 +121,10 @@ class Thread(Messageable, Hashable):
         This is always ``True`` for public threads.
     archiver_id: Optional[:class:`int`]
         The user's ID that archived this thread.
+
+        .. note::
+            Due to an API change, the ``archiver_id`` will always be ``None`` and can only be obtained via the audit log now.
```

Comment:
```suggestion
            Due to an API change, the ``archiver_id`` will always be ``None`` and can only be obtained via the audit log.
```

Good catch, but I forgot to remove this too.